api/envoy/extensions/filters/http/gcp_authn/v3/gcp_authn.proto

```diff
@@ -24,17 +26,37 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 message GcpAuthnFilterConfig {
   // The HTTP URI to fetch tokens from GCE Metadata Server(https://cloud.google.com/compute/docs/metadata/overview).
   // The URL format is "http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/default/identity?audience=[AUDIENCE]"
-  config.core.v3.HttpUri http_uri = 1 [(validate.rules).message = {required: true}];
+  //
+  // This field is deprecated because it does not match the API surface provided by the google auth libraries.
+  // Control planes should not attempt to override the metadata server URI.
+  // The cluster and timeout can be configured using the `cluster` and `timeout` fields instead.
+  // For backward compatibility, the cluster and timeout configured in this field will be used
+  // if the new `cluster` and `timeout` fields are not set.
```

Comment:
One CI failure I noticed is that you will need to have double backticks like `cluster` --> ``cluster``

Reply:
Thanks, fixed.